src/deterministic_vi/elbo_objective.jl

```diff
@@ -36,39 +36,54 @@ function calculate_source_pixel_brightness!{NumType <: Number}(
     @inbounds for i in 1:Ia # Stars and galaxies
         fsm_i = (i == 1) ? fs0m : fs1m
         a_i = ea.vp[s][ids.a[i, 1]]
+        sb_E_l_a_b_i = sb.E_l_a[b, i]
```

Comment:
I haven't tried on this PR. I just tried a few in my own pass through the code last week where I couldn't see an effect.